corehq/motech/repeaters/models.py

```diff
@@ -298,6 +299,28 @@ class Meta:
     def repeater(self):
         return Repeater.get(self.repeater_id)
 
+    def to_json(self):
+        repeater_dict = self.__dict__.copy()
```

Comment:
Is it ever possible for this to contain non-JSON-serializable values (example: datetime values)? Seems like we want a protocol for getting only the attributes/values that should included in the JSON, as well as a way to convert those values if necessary.

Reply:
https://github.com/dimagi/commcare-hq/pull/31792/commits/45b12a907560c11297c97f87192be6bc5e8fda3f